packages/morph/lib/attr-morph.js

```diff
@@ -6,17 +6,20 @@ function updateProperty(value) {
   this.domHelper.setPropertyStrict(this.element, this.attrName, value);
 }
 
-function updateAttribute(value) {
+function updateAttribute(value, namespace) {
```

Comment:
> This would lead to updateAttribute being used for namespaced attributes on svg-namespaced elements.

The `namespace` of attributes should have nothing to do with the element. Why would adding another `_update` interface to `AttrMorph` change anything for svg namespaced elements?